pinot-integration-tests/src/test/java/com/linkedin/pinot/integration/tests/UploadRefreshDeleteIntegrationTest.java

```diff
@@ -153,6 +210,16 @@ public void testRefresh(String tableName, SegmentVersion version) throws Excepti
     verifyNRows(nRows2, nRows2+nRows3);
   }
 
+  @Test(dataProvider = "configProvider")
```

Comment:
This is the specification for dataProvider. See http://www.seleniumeasy.com/testng-tutorials/dataprovider-in-testng for details.
We need dataProvider as well as the arguments which get substituted in each run.